bin/manuscripts

```diff
@@ -36,12 +36,15 @@ from dateutil import parser
 sys.path.insert(0, '.')
 from manuscripts.report import Report
 from manuscripts.config import Config
+from manuscripts._version import __version__
 
 def get_params():
     """Parse command line arguments"""
 
     parser = argparse.ArgumentParser()
 
+    parser.add_argument('-v', '--version', action='version',version=__version__,
```

Comment:
I think `version=`should be `default=`, see https://docs.python.org/3/library/argparse.html#default

If you think it is better as you have it, please let us know why.

Reply:
As I understand it, -v/--version is to be used to check the version information of manuscripts. Wouldn't making it 'default=' instead of 'version=' convert it into a command-line argument which defaults to a value if the argument is not given rather than just giving the version of manuscripts whenever --version is used ? 
I was trying to do something similar to what is given [here](https://docs.python.org/3/library/argparse.html#action)